build.sbt

```diff
@@ -308,8 +308,8 @@ lazy val cli = project
 
             def zipEntries(zipStream: ZipInputStream): Iterator[(ZipEntry, Array[Byte])] =
               new Iterator[(ZipEntry, Array[Byte])] {
-                var nextEntry = Option.empty[ZipEntry]
-                def update() =
+                private var nextEntry = Option.empty[ZipEntry]
+                private def update() =
```

Comment:
(Admittedly, that's kind of nit-picky, I'm fine with keeping the `private`... :-)